playbooks/common/openshift-master/config.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Set master facts and determine if external etcd certs need to be generated
-  hosts: oo_masters_to_config
+  hosts: oo_first_master:oo_masters_to_config
```

Comment:
certs_missing should just create a list to iterate through in openshift_master_certificates